test/__snapshot__/compiler/error-recovery/js/gen/assignment/x007b_foox003a_bar_.md

```diff
@@ -181,3 +181,9 @@ x = {{ foo: bar,}
 
 ```
 
+### Transform
```

Comment:
@aladdin-add  Why "Transform" here in each test? Should be separated into its own '.md.' file with 'transform:true' as option. Same as I did for printer. Different .md template

Something like this

```js
options  

{ transform: true, transformers: [es2020, es2019] }

Input


ES2020 transform

ES 2019 transform

Printed
```

Not every test should include transformation

Reply:
I splitted the options to be:
```md
### parser options
...
### printer options
...
### transform options
...
```

it will be in the output block only if having the options. e.g. if no printer options found in the md, there is no `printed`. thoughts?